### PR TITLE
DR-179: bootstrap scipt

### PR DESCRIPTION
**Pull request by ahaessly**
script to create groups and resource needed to bootstrap the data repository

### CODE REVIEW

File: datarepoBootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Create the ops group
+curl -X POST --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}" 
+# Add the user as an admin of the ops group
+curl -X PUT --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}/admin/${OPS_ADMIN_EMAIL}" 
+
+# Get the email address for the ops group
+OPS_EMAIL=$(curl --header 'Content-Type: application/json' --header 'Accept: application/json' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${OPS_GROUP_NAME}")
+
+# Create the stewards group
+curl -X POST --header 'Content-Length: 0' --header "Authorization: Bearer ${AUTH_TOKEN}" "${SAM_URL}/api/groups/v1/${STEWARDS_GROUP_NAME}"
```
Comment (thathert) [inline datarepoBootstrap.sh:27]: Why the context length 0 piece?

**Review by thathert — APPROVED** [review]
LGTM, the content length thing is throwing me off

File: datarepoBootstrap.sh (hunk 1 above)
Comment (ahaessly) [reply datarepoBootstrap.sh:27]: curl wouldn't work for a POST with no data without it - even though swagger didn't need it

Comment (ahaessly) [reply datarepoBootstrap.sh:27]: i could pass -d {} instead of setting content length.

**Review by ddietterich — APPROVED** [review]
Not sure I know enough about what this is doing to approve it. I don't see anything amiss.

**Pull request merged**